### PR TITLE
chore: link to article for more information on how to configure & deploy schema changes in a CI/CD pipeline

### LINKED ISSUE #2744: Add link to Prisma Migrate example for CI/CD
This following example posted on the blog is a good one for using Prisma Migrate in CI/CD
https://www.prisma.io/blog/backend-prisma-typescript-orm-with-postgresql-deployment-bbba1ps7kip5

Do you know where we can add it to Migrate docs?

I think later we would need a dedicated section and not only a link to the blog post. As it's quite an important part of using Migrate.

Example of user question
https://github.com/prisma/prisma/discussions/11131

### PR DESCRIPTION
**Pull request by ruheni**
Closes #2744

### CODE REVIEW

**Comment by Jolg42** [comment]
The content is a bit dated now, using Node.js v12. Maybe it's better to not mention it?

The most confusing part for me is that this blog post recommends to use `prisma db push --preview-feature` for testing the deployment and `prisma migrate deploy` for deploying to production.
It should be the same (using `migrate deploy`) I think.

https://github.com/2color/real-world-grading-app/blob/master/package.json#L39
https://www.prisma.io/blog/backend-prisma-typescript-orm-with-postgresql-deployment-bbba1ps7kip5#defining-a-workflow-with-github-actions

**Comment by ruheni** [comment]
I'll add a GitHub action example to the page if that helps.

File: content/300-guides/200-deployment/150-deploy-database-changes-with-prisma-migrate.mdx [hunk 1]
```diff
@@ -33,6 +33,32 @@ For more information about the `migrate deploy` command, see:
...
+
+As part of your CI/CD, you can run `prisma migrate deploy` as part of your pipeline to apply pending migrations to your production database.
+
+Here is an example action that deploys schema changes:
+
+```yaml file=deploy.yml highlight=17-20
+name: Deploy
+on:
+  push:
+    branches: -main
```
Comment (Jolg42) [inline content/300-guides/200-deployment/150-deploy-database-changes-with-prisma-migrate.mdx]:
```suggestion
    branches: 
      - main
```

File: content/300-guides/200-deployment/150-deploy-database-changes-with-prisma-migrate.mdx [hunk 2]
```diff
@@ -33,6 +33,32 @@ For more information about the `migrate deploy` command, see:
...
+
+jobs:
+  deploy:
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
```
Comment (Jolg42) [inline content/300-guides/200-deployment/150-deploy-database-changes-with-prisma-migrate.mdx]:
```suggestion
        uses: actions/checkout@v3
```

File: content/300-guides/200-deployment/150-deploy-database-changes-with-prisma-migrate.mdx [hunk 3]
```diff
@@ -33,6 +33,32 @@ For more information about the `migrate deploy` command, see:
...
+      - name: Setup Node
+        uses: actions/setup-node@v3
+      - name: Install dependencies
+        run: npm install
+      - run: npm run build
+      - name: Run production migration
```
Comment (Jolg42) [inline content/300-guides/200-deployment/150-deploy-database-changes-with-prisma-migrate.mdx]:
```suggestion
      - name: Apply all pending migrations to the database
```

File: content/300-guides/200-deployment/150-deploy-database-changes-with-prisma-migrate.mdx [hunk 4]
```diff
@@ -33,6 +33,32 @@ For more information about the `migrate deploy` command, see:
...
 
-For more information on how to deploy database changes in a CI/CD pipeline, see [Backend with TypeScript, PostgreSQL & Prisma: CI & Deployment](https://www.prisma.io/blog/backend-prisma-typescript-orm-with-postgresql-deployment-bbba1ps7kip5).
-
 </TopBlock>
+
+## Deploying database changes using GitHub Actions
+
+As part of your CI/CD, you can run `prisma migrate deploy` as part of your pipeline to apply pending migrations to your production database.
+
+Here is an example action that deploys schema changes:
```
Comment (Jolg42) [inline content/300-guides/200-deployment/150-deploy-database-changes-with-prisma-migrate.mdx]:
```suggestion
Here is an example action that will run your migrations against your database:
```

**Pull request merged**